packages/hash/frontend/src/components/layout/PageHeader/SearchBar/index.tsx

```diff
@@ -56,6 +64,7 @@ const ResultList: React.FC<{
     component="ul"
     sx={(theme) => ({
       position: !isMobile ? "absolute" : "unset",
+      top: !isMobile ? "calc(100% + 1px)" : "unset",
```

Comment:
The search result list currently doesn't match the designs, but I assume this is to be addressed in follow-up. Just want to make sure there's a task?


Currently:
<img width="441" alt="image" src="https://user-images.githubusercontent.com/42802102/158166037-fc4351ae-faec-4d67-aa8f-629fc04823e0.png">
Should be:
<img width="451" alt="image" src="https://user-images.githubusercontent.com/42802102/158166084-81cf4838-2508-4cb7-872a-ce891ee6cbe3.png">

Also I think we should make use of the MUI List component here, just as we are in the other dropdown menus.

Reply:
This design wasn't present while this component was being built. It makes sense to take this up in a follow-up to prevent further delaying this PR (captured [here](https://app.asana.com/0/1200211978612931/1201961850830772/f)).

Can you also add a link to the results design in this thread as I can't find it on Figma myself.

Reply:
Sure, I think it's this: https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-Editor?node-id=179%3A4921